arpeggio/__init__.py

```diff
@@ -779,19 +779,27 @@ class RegExMatch(Match):
             It will be used to create regular expression using re.compile.
         ignore_case(bool): If case insensitive match is needed.
             Default is None to support propagation from global parser setting.
+        multiline(bool): allow regex to works on multiple lines (re.DOTALL flag)
         str_repr(str): A string that is used to represent this regex.
+        re_flags: flags parameter for re.compile if neither ignore_case
+            or multiple are set.
 
     '''
     def __init__(self, to_match, rule_name='', root=False, ignore_case=None,
-                 str_repr=None):
+                 multiline=False, str_repr=None, re_flags=re.MULTILINE):
```

Comment:
`multiline` param should by default be `None` to test if the user set it explicitly.